js-exercises/accessor-properties/accessorProperties.js

```diff
@@ -0,0 +1,28 @@
+function accessorProperties() {
+  const newObject = {};
+  Object.defineProperties(newObject, {
+    _number: {
+      value: null,
+      writable: true
+    },
+    number: {
```

Comment:
This '_number' seems to a bit redundant here. One suggestion would be to do something like this:
Object.defineProperty(obj, 'number', {
    configurable: true,
    enumerable: true,
    get: function() {
      if(!isNaN(this.value)){}
      ...../